ddtrace/llmobs/utils.py

```diff
@@ -19,7 +19,18 @@
 ExportedLLMObsSpan = TypedDict("ExportedLLMObsSpan", {"span_id": str, "trace_id": str})
 Document = TypedDict("Document", {"name": str, "id": str, "text": str, "score": float}, total=False)
 Message = TypedDict("Message", {"content": str, "role": str}, total=False)
-Prompt = TypedDict("Prompt", {"variables": Dict[str, str], "template": str, "id": str, "version": str}, total=False)
+Prompt = TypedDict(
+    "Prompt",
+    {
+        "variables": Dict[str, str],
+        "template": str,
+        "id": str,
+        "version": str,
+        "rag_context_variables": List[str],
+        "rag_query_variables": str,
```

Comment:
```suggestion
        "rag_query_variables": List[str],
```
is `rag_query_variables` a `str` or `List[str]`? in `validate_prompt`, it looks like it's treated as a `List[str]` in validation, although when grabbing it from the `prompt` it's named as a singular `rag_query_variable_key`, suggesting it is a singular string

Reply:
It's meant to be a `List[str]` - thanks for catching, i've updated the type annotation and renamed `rag_query_variable_key` to plural